test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -510,15 +540,13 @@ static int open_pktios(void)
 		if (rc != 0)
 			printf("****** promisc_mode_set failed  ******\n");
 
-		pktios[iface] = pktio;
-
 		if (odp_pktin_queue(pktio, &pktins[iface], 1) != 1) {
 			odp_pktio_close(pktio);
 			ODPH_ERR("odp_pktio_open() failed: no pktin queue\n");
 			return -1;
 		}
 
-		if (odp_pktout_queue(pktio, &pktouts[iface], 1) != 1) {
+		if (iface && odp_pktout_queue(pktio, &pktouts[iface], 1) != 1) {
```

Comment:
This could be just removed, along with the odp_pktout_queue_config() that is done a bit earlier. The API says: "When packet output mode is ODP_PKTOUT_MODE_DISABLED or ODP_PKTOUT_MODE_TM, odp_pktout_queue_config() call is optional and will ignore all parameters."